content/tutorials/presence.textile

```diff
@@ -313,10 +417,10 @@ blang[ruby].
 h2. Next steps
 
 1. If you would like to find out more about how channels, publishing and subscribing works, see the "Realtime channels":/realtime/channels & "messages":/realtime/messages documentation
-1. If you would like to find out more about how presence works, see the "Realtime presence documentation":/realtime/presence
-2. Learn more about "Ably features":https://ably.com/platform by stepping through our other "Ably tutorials":https://ably.com/tutorials
-3. Gain a good technical "overview of how the Ably realtime platform works":https://ably.com/docs/key-concepts
-4. "Get in touch if you need help":https://ably.com/contact
+2. If you would like to find out more about how presence works, see the "Realtime presence documentation":/realtime/presence
+3. Learn more about "Ably features":https://ably.com/platform by stepping through our other "Ably tutorials":https://ably.com/tutorials
+4. Gain a good technical overview of "how the Ably realtime platform works":https://ably.com/docs/key-concepts
+5. "Get in touch if you need help":https://ably.com/contact
```

Comment:
i think these links do not use the full domain name ie. `https://ably.com` just the relative path. (see #1 and #2)
(that is what i noticed on other PRs)

Reply:
Good spot! Done